app-modules/notification/src/Actions/MakeOutboundDeliverable.php

```diff
@@ -49,22 +48,20 @@
 use AdvisingApp\Notification\Notifications\Channels\EmailChannel;
 use AdvisingApp\Notification\Notifications\Channels\DatabaseChannel;
 
-class CreateOutboundDeliverable
+class MakeOutboundDeliverable
 {
     public function handle(BaseNotification $notification, object $notifiable, string $channel): OutboundDeliverable
     {
         $channel = match ($channel) {
             SmsChannel::class => NotificationChannel::Sms,
             EmailChannel::class => NotificationChannel::Email,
             DatabaseChannel::class => NotificationChannel::Database,
-            default => throw new Exception('Invalid notification channel.'),
```

Comment:
Is there a reason we removed the `default` cases for the two match statements in this method?

Reply:
I believe sonar was complaining about them.

Reply:
That's a bit odd, would think it would complain the other way around 🤔

Reply:
I'm fine putting them back for now if you want?